### DISCUSSION TITLE
Issue in .net 6

### DISCUSSION THREAD

**Discussion by Tonsterr** [cefsharp/CefSharp#4273] [Q&A] [answered]
Hi,

I'm converting a project from .net framework 4.7.2 to .net 6. I uninstalled the old cefshap nuget and installed the latest cefsharp.core nuget (106.0.260) instead, but now i'm running into an issue.  Previously i had this code

`ViewBrowser.JavascriptObjectRepository.Register("bound", myObject, false);`

but in the latest cefsharp core package, the last parameter 'false', which was the isasync parameter, is no longer accepted. So i changed my code to this:

`ViewBrowser.JavascriptObjectRepository.Register("bound", myObject);`

However, now my page is not loading properly.

So 3 questions I have are: 1) how come the isasync parameter was removed in the .core version? 2) What does the Register function default to now? isasync = true? 3) any input on what i'm doing wrong that could resolve my issue?

**Comment by amaitland** [comment, marked as answer]
The synchronous JavaScript binding implementation uses WCF which wasn't ported to .Net 5/6/7 by Microsoft. Only the asynchronous JavaScript binding implementation which uses chromium IPC is supported. 

For details of async JavaScript binding see https://github.com/cefsharp/CefSharp/wiki/General-Usage#async-javascript-binding-jsb

**Reply by Tonsterr** [reply]
Ah ok i get it now. Thanks for the help.
